src/RazorSdk/Targets/Microsoft.NET.Sdk.Razor.Component.targets

```diff
@@ -22,7 +22,6 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <PropertyGroup>
     <_RazorGenerateComponentDeclarationDesignTimeDependsOn>ResolveRazorConfiguration;ResolveRazorComponentInputs;AssignRazorComponentTargetPaths;RazorGenerateComponentDeclaration</_RazorGenerateComponentDeclarationDesignTimeDependsOn>
```

Comment:
I'm not moving this since it actually relies on one the code generation targets that no longer is imported in 6.0 projects. However turns out that this design time target is no longer being used. We could consider removing this or leave this here if we need to continue support older versions of tools,